src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js

```diff
@@ -108,30 +108,49 @@ class FilterBar extends React.Component {
     this.props.applyFilters(this.props.checked);
   }
 
+  /**
+   * Check if applying a filter results in content or not
+   * @param {string} filterGroup Key in filterCounts where the filter can be found, e.g. 'language'
+   * @param {string} filterItem The filter to check, e.g. 'nb'
+   * @return {boolean} true if applying filter option will result in content, false if it results in zero content
+   */
+  filterHasContent = (filterGroup, filterItem) => {
+    const counts = this.state.filterCounts[filterGroup] ?? [];
+    return counts[filterItem] > 0 ?? false;
```

Comment:
`??` doesn't do anything in this context. For a non-existent item, this is equivalent to `(undefined > 0) ?? false`, and comparing undefined against a number returns false, so the expression to the right of `??` is never evaluated.

Reply:
Fixed